sql/src/main/java/org/apache/druid/sql/calcite/planner/QueryHandler.java

```diff
@@ -684,16 +684,7 @@ private DruidException buildSQLPlanningError(RelOptPlanner.CannotPlanException e
                           .ofCategory(DruidException.Category.UNSUPPORTED)
                           .build(exception, "Unhandled Query Planning Failure, see broker logs for details");
     } else {
-      // Planning errors are more like hints: it isn't guaranteed that the planning error is actually what went wrong.
-      // For this reason, we consider these as targetting a more expert persona, i.e. the admin instead of the actual
-      // user.
```

Comment:
The removed comment provides the reason why it was not target to the `USER` ; I think we should leave the uncategorized ones to `ADMIN` or something

if there is a particular set of exceptions which should be hardened; maybe those should be targeted directly

Reply:
I reverted this part of the change, thanks for taking a look!